dZoneManager/Level.cpp

```diff
@@ -193,13 +193,14 @@ void Level::ReadSceneObjectDataChunk(std::ifstream & file, Header & header) {
             ldfString.push_back(data);
         }
         
-        std::string sData = GeneralUtils::UTF16ToWTF8(ldfString);
+        std::string sData = GeneralUtils::UTF16ToUTF8(ldfString);
         std::stringstream ssData(sData);
         std::string token;
         char deliminator = '\n';
         
         while (std::getline(ssData, token, deliminator)) {
             LDFBaseData * ldfData = LDFBaseData::DataFromString(token);
+			if (!ldfData) continue;
```

Comment:
Why this change?

Reply:
Oh, that was a temp change I thought I fixed. I still need to work that out. My bad.